{{ cookiecutter.project_slug }}/meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: "An ansible role for {{ cookiecutter.project_short_purpose }}."
 
   author: "{{ cookiecutter.galaxy_name }}"
-  license: MIT
+  license: "{{ cookiecutter.open_source_license }}"
```

Comment:
https://docs.ansible.com/ansible/latest/dev_guide/collections_galaxy_meta.html

> Ansible Galaxy currently only accepts [SPDX](https://spdx.org/licenses/) licenses

Reply:
resolved in [0d39515](https://github.com/JonasPammer/cookiecutter-ansible-role/pull/34/commits/0d3951534ee73a8993a7ce62fbaac14b8d321a12)